blank-canvas-blocks/assets/navigation.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * File navigation.js.
+ *
+ * Required to open the mobile navigation.
+ */
+(function () {
+
+	function mobalizeNavigationBlock( navMenu ) {
+		if( !navMenu ){
+			return;
+		}
+		addMobileMenuOpenButton( navMenu );
+		addMobileMenuCloseButton( navMenu );
+	}
+
+	function addMobileMenuOpenButton( navMenu ) {
+		const menuContainer = navMenu.querySelector( '.wp-block-navigation__container' );
+		const openButton = document.createElement( 'button' );
+		const openButtonLabel = getComputedStyle(menuContainer).getPropertyValue( '--wp--custom--navigation--mobile--menu--open-label' );
+		openButton.classList.add( 'wp-block-navigation__mobile-menu-open-button' );
+		openButton.innerText = openButtonLabel;
+		openButton.addEventListener( 'click', (clickEvent) => {
+			console.log('clickEvent', clickEvent);
+			navMenu.classList.add( 'show' );
+			menuContainer.scrollTop = 0;
+			if( 0 === clickEvent.detail) {
+				// Menu was opened with keyboard, apply focus to close button.
+				menuContainer.querySelector('button:first-of-type')?.focus();
```

Comment:
To confirm, we can use optional chaining (`?.`) because we are _not_ supporting IE, correct?

Reply:
Oh, man, force of habit.  Previously I ran all JS code through Babel to transpile it for IE.  
None of this would work in IE though since it's so dependent on CSS Variables.  Easy enough to add a null check instead though in case the CSS vars polyfill can do the trick here.

Reply:
Refactored out the cool stuff.
Replaced with boring 'ol null checks.